hathor/websocket/factory.py

```diff
@@ -271,6 +287,8 @@ def handle_message(self, connection: HathorAdminWebsocketProtocol, data: bytes)
             self._handle_subscribe_address(connection, message)
         elif message['type'] == 'unsubscribe_address':
             self._handle_unsubscribe_address(connection, message)
+        elif message['type'] == 'subscribe':
```

Comment:
Should we also have an unsubscribe message? Doesn't seem we'll use it now, but might be useful in the future.

Reply:
Done